gschem/src/x_autonumber.c

```diff
@@ -975,7 +985,7 @@ AUTONUMBER_TEXT *autonumber_init_state()
   if(autotext==NULL) return NULL;
 
   autotext->scope_text = NULL;
-  t=default_text;
+  t = (gchar**) default_text;
```

Comment:
Since `t` is an offset into `default_text`, I suspect it should be `const gchar **t`.